python/tank/platform/environment.py

```diff
@@ -329,8 +328,24 @@ def get_app_descriptor(self, engine_name, app_name):
 
         return d
 
-    def set_context(self, context):
```

Comment:
will this break something? Sometimes client code uses methods that we consider internal. What's the rationale of renameing set_context to change_context? Just to make things clearer? I wonder if in this case we could just keep the current name for simplicity?